[2017-11-24T10:48:21.878] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:43:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T11:13:50.165] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:45:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T11:18:40.674] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:45:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T12:06:33.022] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:45:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T13:34:17.618] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:45:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T13:46:34.711] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:45:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T13:59:52.203] [ERROR] app - Something went wrong: { TypeError: F:\project\project\blog\views\content.ejs:4
    2| <html>
    3| <head>
 >> 4|     <title><%= content.title%></title>
    5|     <meta content="<%= content.description%>" name="description">
    6|     <meta content="<%= content.keyword%>" name="keywords">
    7| 

Cannot read property 'title' of undefined
    at eval (eval at compile (F:\project\project\blog\node_modules\ejs\lib\ejs.js:549:12), <anonymous>:11:34)
    at returnedFn (F:\project\project\blog\node_modules\ejs\lib\ejs.js:580:17)
    at tryHandleCache (F:\project\project\blog\node_modules\ejs\lib\ejs.js:223:34)
    at View.exports.renderFile [as engine] (F:\project\project\blog\node_modules\ejs\lib\ejs.js:437:10)
    at View.render (F:\project\project\blog\node_modules\express\lib\view.js:127:8)
    at tryRender (F:\project\project\blog\node_modules\express\lib\application.js:640:10)
    at Function.render (F:\project\project\blog\node_modules\express\lib\application.js:592:3)
    at ServerResponse.render (F:\project\project\blog\node_modules\express\lib\response.js:971:7)
    at F:\project\project\blog\routes\content.js:46:9
    at <anonymous> path: 'F:\\project\\project\\blog\\views\\content.ejs' }
[2017-11-24T14:00:22.611] [ERROR] app - Something went wrong: { TypeError: F:\project\project\blog\views\content.ejs:4
    2| <html>
    3| <head>
 >> 4|     <title><%= content.title%></title>
    5|     <meta content="<%= content.description%>" name="description">
    6|     <meta content="<%= content.keyword%>" name="keywords">
    7| 

Cannot read property 'title' of undefined
    at eval (eval at compile (F:\project\project\blog\node_modules\ejs\lib\ejs.js:549:12), <anonymous>:11:34)
    at returnedFn (F:\project\project\blog\node_modules\ejs\lib\ejs.js:580:17)
    at tryHandleCache (F:\project\project\blog\node_modules\ejs\lib\ejs.js:223:34)
    at View.exports.renderFile [as engine] (F:\project\project\blog\node_modules\ejs\lib\ejs.js:437:10)
    at View.render (F:\project\project\blog\node_modules\express\lib\view.js:127:8)
    at tryRender (F:\project\project\blog\node_modules\express\lib\application.js:640:10)
    at Function.render (F:\project\project\blog\node_modules\express\lib\application.js:592:3)
    at ServerResponse.render (F:\project\project\blog\node_modules\express\lib\response.js:971:7)
    at F:\project\project\blog\routes\content.js:46:9
    at <anonymous> path: 'F:\\project\\project\\blog\\views\\content.ejs' }
[2017-11-24T14:01:27.214] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:45:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T15:15:20.179] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:45:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T15:17:00.468] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:45:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T15:32:39.624] [ERROR] app - { Error: ER_DUP_ENTRY: Duplicate entry 'c2620d0d-8b64-40d7-b83f-405df34a0003' for key 'PRIMARY'
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:186:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'c2620d0d-8b64-40d7-b83f-405df34a0003\' for key \'PRIMARY\'',
  sqlState: '23000',
  index: 0,
  sql: 'UPDATE cms SET id=\'c2620d0d-8b64-40d7-b83f-405df34a0003\' , title=\'标题\',keyword=\'测试\',description=\'测试\',content=\'<p>标题11</p>\',menu=\'js\',isindex=\'1\'' }
[2017-11-24T15:32:59.911] [ERROR] app - Something went wrong: { Error: ER_DUP_ENTRY: Duplicate entry 'c2620d0d-8b64-40d7-b83f-405df34a0003' for key 'PRIMARY'
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:186:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'c2620d0d-8b64-40d7-b83f-405df34a0003\' for key \'PRIMARY\'',
  sqlState: '23000',
  index: 0,
  sql: 'UPDATE cms SET id=\'c2620d0d-8b64-40d7-b83f-405df34a0003\' , title=\'标题\',keyword=\'测试\',description=\'测试\',content=\'<p>标题11</p>\',menu=\'js\',isindex=\'1\'' }
[2017-11-24T15:33:02.989] [ERROR] app - Something went wrong: { TypeError: F:\project\project\blog\views\content.ejs:4
    2| <html>
    3| <head>
 >> 4|     <title><%= content.title%></title>
    5|     <meta content="<%= content.description%>" name="description">
    6|     <meta content="<%= content.keyword%>" name="keywords">
    7| 

Cannot read property 'title' of undefined
    at eval (eval at compile (F:\project\project\blog\node_modules\ejs\lib\ejs.js:549:12), <anonymous>:11:34)
    at returnedFn (F:\project\project\blog\node_modules\ejs\lib\ejs.js:580:17)
    at tryHandleCache (F:\project\project\blog\node_modules\ejs\lib\ejs.js:223:34)
    at View.exports.renderFile [as engine] (F:\project\project\blog\node_modules\ejs\lib\ejs.js:437:10)
    at View.render (F:\project\project\blog\node_modules\express\lib\view.js:127:8)
    at tryRender (F:\project\project\blog\node_modules\express\lib\application.js:640:10)
    at Function.render (F:\project\project\blog\node_modules\express\lib\application.js:592:3)
    at ServerResponse.render (F:\project\project\blog\node_modules\express\lib\response.js:971:7)
    at F:\project\project\blog\routes\content.js:46:9
    at <anonymous> path: 'F:\\project\\project\\blog\\views\\content.ejs' }
[2017-11-24T15:33:48.252] [ERROR] app - { Error: ER_DUP_ENTRY: Duplicate entry 'c2620d0d-8b64-40d7-b83f-405df34a0003' for key 'PRIMARY'
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:186:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'c2620d0d-8b64-40d7-b83f-405df34a0003\' for key \'PRIMARY\'',
  sqlState: '23000',
  index: 0,
  sql: 'UPDATE cms SET id=\'c2620d0d-8b64-40d7-b83f-405df34a0003\' , title=\'标题11\',keyword=\'测试\',description=\'测试\',content=\'<p>标题</p>\',menu=\'js\',isindex=\'1\'' }
[2017-11-24T15:33:48.258] [ERROR] app - Something went wrong: { Error: ER_DUP_ENTRY: Duplicate entry 'c2620d0d-8b64-40d7-b83f-405df34a0003' for key 'PRIMARY'
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:186:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_DUP_ENTRY',
  errno: 1062,
  sqlMessage: 'Duplicate entry \'c2620d0d-8b64-40d7-b83f-405df34a0003\' for key \'PRIMARY\'',
  sqlState: '23000',
  index: 0,
  sql: 'UPDATE cms SET id=\'c2620d0d-8b64-40d7-b83f-405df34a0003\' , title=\'标题11\',keyword=\'测试\',description=\'测试\',content=\'<p>标题</p>\',menu=\'js\',isindex=\'1\'' }
[2017-11-24T15:33:51.304] [ERROR] app - Something went wrong: { TypeError: F:\project\project\blog\views\content.ejs:4
    2| <html>
    3| <head>
 >> 4|     <title><%= content.title%></title>
    5|     <meta content="<%= content.description%>" name="description">
    6|     <meta content="<%= content.keyword%>" name="keywords">
    7| 

Cannot read property 'title' of undefined
    at eval (eval at compile (F:\project\project\blog\node_modules\ejs\lib\ejs.js:549:12), <anonymous>:11:34)
    at returnedFn (F:\project\project\blog\node_modules\ejs\lib\ejs.js:580:17)
    at tryHandleCache (F:\project\project\blog\node_modules\ejs\lib\ejs.js:223:34)
    at View.exports.renderFile [as engine] (F:\project\project\blog\node_modules\ejs\lib\ejs.js:437:10)
    at View.render (F:\project\project\blog\node_modules\express\lib\view.js:127:8)
    at tryRender (F:\project\project\blog\node_modules\express\lib\application.js:640:10)
    at Function.render (F:\project\project\blog\node_modules\express\lib\application.js:592:3)
    at ServerResponse.render (F:\project\project\blog\node_modules\express\lib\response.js:971:7)
    at F:\project\project\blog\routes\content.js:46:9
    at <anonymous> path: 'F:\\project\\project\\blog\\views\\content.ejs' }
[2017-11-24T15:43:03.029] [ERROR] app - { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''10'' at line 1
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:39:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'\'10\'\' at line 1',
  sqlState: '42000',
  index: 0,
  sql: 'SELECT id,title,keyword,description,date,menu FROM  cms where del!=1 and menu=\'js\' ORDER BY date DESC ,isindex DESC limit 1,\'10\' ' }
[2017-11-24T15:43:03.035] [ERROR] app - Something went wrong: { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''10'' at line 1
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:39:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'\'10\'\' at line 1',
  sqlState: '42000',
  index: 0,
  sql: 'SELECT id,title,keyword,description,date,menu FROM  cms where del!=1 and menu=\'js\' ORDER BY date DESC ,isindex DESC limit 1,\'10\' ' }
[2017-11-24T15:43:04.590] [ERROR] app - { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''10'' at line 1
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:39:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'\'10\'\' at line 1',
  sqlState: '42000',
  index: 0,
  sql: 'SELECT id,title,keyword,description,date,menu FROM  cms where del!=1 and menu=\'js\' ORDER BY date DESC ,isindex DESC limit 1,\'10\' ' }
[2017-11-24T15:43:04.591] [ERROR] app - Something went wrong: { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''10'' at line 1
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:39:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'\'10\'\' at line 1',
  sqlState: '42000',
  index: 0,
  sql: 'SELECT id,title,keyword,description,date,menu FROM  cms where del!=1 and menu=\'js\' ORDER BY date DESC ,isindex DESC limit 1,\'10\' ' }
[2017-11-24T15:44:01.166] [ERROR] app - { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''10'' at line 1
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:39:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'\'10\'\' at line 1',
  sqlState: '42000',
  index: 0,
  sql: 'SELECT id,title,keyword,description,date,menu FROM  cms where del!=1 and menu=\'js\' ORDER BY date DESC ,isindex DESC limit 1,\'10\'' }
[2017-11-24T15:44:01.172] [ERROR] app - Something went wrong: { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''10'' at line 1
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:39:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'\'10\'\' at line 1',
  sqlState: '42000',
  index: 0,
  sql: 'SELECT id,title,keyword,description,date,menu FROM  cms where del!=1 and menu=\'js\' ORDER BY date DESC ,isindex DESC limit 1,\'10\'' }
[2017-11-24T15:44:02.470] [ERROR] app - { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''10'' at line 1
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:39:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'\'10\'\' at line 1',
  sqlState: '42000',
  index: 0,
  sql: 'SELECT id,title,keyword,description,date,menu FROM  cms where del!=1 and menu=\'js\' ORDER BY date DESC ,isindex DESC limit 1,\'10\'' }
[2017-11-24T15:44:02.471] [ERROR] app - Something went wrong: { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''10'' at line 1
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:39:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'\'10\'\' at line 1',
  sqlState: '42000',
  index: 0,
  sql: 'SELECT id,title,keyword,description,date,menu FROM  cms where del!=1 and menu=\'js\' ORDER BY date DESC ,isindex DESC limit 1,\'10\'' }
[2017-11-24T15:44:55.231] [ERROR] app - { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''10'' at line 1
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:39:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'\'10\'\' at line 1',
  sqlState: '42000',
  index: 0,
  sql: 'SELECT id,title,keyword,description,date,menu FROM  cms where del!=1 and menu=\'js\' ORDER BY date DESC ,isindex DESC limit 1,\'10\'' }
[2017-11-24T15:44:55.231] [ERROR] app - Something went wrong: { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''10'' at line 1
    at Query.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at PoolConnection.query (F:\project\project\blog\node_modules\mysql\lib\Connection.js:208:25)
    at F:\project\project\blog\dao\cms.js:39:24
    at Ping.onOperationComplete [as _callback] (F:\project\project\blog\node_modules\mysql\lib\Pool.js:110:5)
    at Ping.Sequence.end (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:88:24)
    at Ping.Sequence.OkPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:97:8)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlMessage: 'You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near \'\'10\'\' at line 1',
  sqlState: '42000',
  index: 0,
  sql: 'SELECT id,title,keyword,description,date,menu FROM  cms where del!=1 and menu=\'js\' ORDER BY date DESC ,isindex DESC limit 1,\'10\'' }
[2017-11-24T15:52:45.169] [ERROR] app - Something went wrong: { SyntaxError: Unexpected token # in JSON at position 0
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:157:10)
    at parse (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:83:15)
    at F:\project\project\blog\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (F:\project\project\blog\node_modules\raw-body\index.js:224:16)
    at done (F:\project\project\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (F:\project\project\blog\node_modules\raw-body\index.js:273:7)
    at emitNone (events.js:115:13)
    at IncomingMessage.emit (events.js:218:7)
    at endReadableNT (_stream_readable.js:1054:12)
  expose: true,
  statusCode: 400,
  status: 400,
  body: '83aea0de-dde1-474b-bc40-edb5a316f7d4',
  type: 'entity.parse.failed' }
[2017-11-24T15:55:48.494] [ERROR] app - Something went wrong: { SyntaxError: Unexpected token # in JSON at position 0
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:157:10)
    at parse (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:83:15)
    at F:\project\project\blog\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (F:\project\project\blog\node_modules\raw-body\index.js:224:16)
    at done (F:\project\project\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (F:\project\project\blog\node_modules\raw-body\index.js:273:7)
    at emitNone (events.js:115:13)
    at IncomingMessage.emit (events.js:218:7)
    at endReadableNT (_stream_readable.js:1054:12)
  expose: true,
  statusCode: 400,
  status: 400,
  body: 'c2620d0d-8b64-40d7-b83f-405df34a0003',
  type: 'entity.parse.failed' }
[2017-11-24T15:56:26.001] [ERROR] app - Something went wrong: { SyntaxError: Unexpected token # in JSON at position 0
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:157:10)
    at parse (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:83:15)
    at F:\project\project\blog\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (F:\project\project\blog\node_modules\raw-body\index.js:224:16)
    at done (F:\project\project\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (F:\project\project\blog\node_modules\raw-body\index.js:273:7)
    at emitNone (events.js:115:13)
    at IncomingMessage.emit (events.js:218:7)
    at endReadableNT (_stream_readable.js:1054:12)
  expose: true,
  statusCode: 400,
  status: 400,
  body: 'c2620d0d-8b64-40d7-b83f-405df34a0003',
  type: 'entity.parse.failed' }
[2017-11-24T15:57:44.626] [ERROR] app - Something went wrong: { SyntaxError: Unexpected token # in JSON at position 0
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:157:10)
    at parse (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:83:15)
    at F:\project\project\blog\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (F:\project\project\blog\node_modules\raw-body\index.js:224:16)
    at done (F:\project\project\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (F:\project\project\blog\node_modules\raw-body\index.js:273:7)
    at emitNone (events.js:115:13)
    at IncomingMessage.emit (events.js:218:7)
    at endReadableNT (_stream_readable.js:1054:12)
  expose: true,
  statusCode: 400,
  status: 400,
  body: 'c2620d0d-8b64-40d7-b83f-405df34a0003',
  type: 'entity.parse.failed' }
[2017-11-24T15:58:09.375] [ERROR] app - Something went wrong: { SyntaxError: Unexpected token # in JSON at position 0
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:157:10)
    at parse (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:83:15)
    at F:\project\project\blog\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (F:\project\project\blog\node_modules\raw-body\index.js:224:16)
    at done (F:\project\project\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (F:\project\project\blog\node_modules\raw-body\index.js:273:7)
    at emitNone (events.js:115:13)
    at IncomingMessage.emit (events.js:218:7)
    at endReadableNT (_stream_readable.js:1054:12)
  expose: true,
  statusCode: 400,
  status: 400,
  body: 'c2620d0d-8b64-40d7-b83f-405df34a0003',
  type: 'entity.parse.failed' }
[2017-11-24T15:59:06.255] [ERROR] app - Something went wrong: { SyntaxError: Unexpected token # in JSON at position 0
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:157:10)
    at parse (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:83:15)
    at F:\project\project\blog\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (F:\project\project\blog\node_modules\raw-body\index.js:224:16)
    at done (F:\project\project\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (F:\project\project\blog\node_modules\raw-body\index.js:273:7)
    at emitNone (events.js:115:13)
    at IncomingMessage.emit (events.js:218:7)
    at endReadableNT (_stream_readable.js:1054:12)
  expose: true,
  statusCode: 400,
  status: 400,
  body: 'c2620d0d-8b64-40d7-b83f-405df34a0003',
  type: 'entity.parse.failed' }
[2017-11-24T15:59:47.226] [ERROR] app - Something went wrong: { SyntaxError: Unexpected token # in JSON at position 0
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:157:10)
    at parse (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:83:15)
    at F:\project\project\blog\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (F:\project\project\blog\node_modules\raw-body\index.js:224:16)
    at done (F:\project\project\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (F:\project\project\blog\node_modules\raw-body\index.js:273:7)
    at emitNone (events.js:115:13)
    at IncomingMessage.emit (events.js:218:7)
    at endReadableNT (_stream_readable.js:1054:12)
  expose: true,
  statusCode: 400,
  status: 400,
  body: 'c2620d0d-8b64-40d7-b83f-405df34a0003',
  type: 'entity.parse.failed' }
[2017-11-24T16:00:04.392] [ERROR] app - Something went wrong: { SyntaxError: Unexpected token # in JSON at position 0
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:157:10)
    at parse (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:83:15)
    at F:\project\project\blog\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (F:\project\project\blog\node_modules\raw-body\index.js:224:16)
    at done (F:\project\project\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (F:\project\project\blog\node_modules\raw-body\index.js:273:7)
    at emitNone (events.js:115:13)
    at IncomingMessage.emit (events.js:218:7)
    at endReadableNT (_stream_readable.js:1054:12)
  expose: true,
  statusCode: 400,
  status: 400,
  body: 'c2620d0d-8b64-40d7-b83f-405df34a0003',
  type: 'entity.parse.failed' }
[2017-11-24T16:00:18.568] [ERROR] app - Something went wrong: { SyntaxError: Unexpected token # in JSON at position 0
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:157:10)
    at parse (F:\project\project\blog\node_modules\body-parser\lib\types\json.js:83:15)
    at F:\project\project\blog\node_modules\body-parser\lib\read.js:121:18
    at invokeCallback (F:\project\project\blog\node_modules\raw-body\index.js:224:16)
    at done (F:\project\project\blog\node_modules\raw-body\index.js:213:7)
    at IncomingMessage.onEnd (F:\project\project\blog\node_modules\raw-body\index.js:273:7)
    at emitNone (events.js:115:13)
    at IncomingMessage.emit (events.js:218:7)
    at endReadableNT (_stream_readable.js:1054:12)
  expose: true,
  statusCode: 400,
  status: 400,
  body: 'c2620d0d-8b64-40d7-b83f-405df34a0003',
  type: 'entity.parse.failed' }
[2017-11-24T16:41:32.996] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T16:42:54.591] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T16:43:31.039] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T17:02:20.144] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T17:08:58.773] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-24T17:24:01.489] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-27T10:21:51.663] [ERROR] app - { Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'xxx'@'localhost' (using password: YES)
    at Handshake.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Handshake.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Handshake.js:130:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at Protocol.handshake (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:52:23)
    at PoolConnection.connect (F:\project\project\blog\node_modules\mysql\lib\Connection.js:130:18)
    at Pool.getConnection (F:\project\project\blog\node_modules\mysql\lib\Pool.js:48:16)
    at Promise (F:\project\project\blog\dao\cms.js:34:12)
    at new Promise (<anonymous>)
    at Object.exports.selectList (F:\project\project\blog\dao\cms.js:11:19)
    at Promise (F:\project\project\blog\routes\index.js:31:41)
    at new Promise (<anonymous>)
    at F:\project\project\blog\routes\index.js:30:27
  code: 'ER_ACCESS_DENIED_ERROR',
  errno: 1045,
  sqlMessage: 'Access denied for user \'xxx\'@\'localhost\' (using password: YES)',
  sqlState: '28000',
  fatal: true }
[2017-11-27T10:21:51.675] [ERROR] app - Something went wrong: { Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'xxx'@'localhost' (using password: YES)
    at Handshake.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Handshake.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Handshake.js:130:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at Protocol.handshake (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:52:23)
    at PoolConnection.connect (F:\project\project\blog\node_modules\mysql\lib\Connection.js:130:18)
    at Pool.getConnection (F:\project\project\blog\node_modules\mysql\lib\Pool.js:48:16)
    at Promise (F:\project\project\blog\dao\cms.js:34:12)
    at new Promise (<anonymous>)
    at Object.exports.selectList (F:\project\project\blog\dao\cms.js:11:19)
    at Promise (F:\project\project\blog\routes\index.js:31:41)
    at new Promise (<anonymous>)
    at F:\project\project\blog\routes\index.js:30:27
  code: 'ER_ACCESS_DENIED_ERROR',
  errno: 1045,
  sqlMessage: 'Access denied for user \'xxx\'@\'localhost\' (using password: YES)',
  sqlState: '28000',
  fatal: true }
[2017-11-27T10:21:51.740] [ERROR] app - { Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'xxx'@'localhost' (using password: YES)
    at Handshake.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Handshake.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Handshake.js:130:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at Protocol.handshake (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:52:23)
    at PoolConnection.connect (F:\project\project\blog\node_modules\mysql\lib\Connection.js:130:18)
    at Pool.getConnection (F:\project\project\blog\node_modules\mysql\lib\Pool.js:48:16)
    at Promise (F:\project\project\blog\dao\cms.js:78:12)
    at new Promise (<anonymous>)
    at Object.exports.selectListTotal (F:\project\project\blog\dao\cms.js:63:19)
    at Promise (F:\project\project\blog\routes\index.js:41:42)
    at new Promise (<anonymous>)
    at menu (F:\project\project\blog\routes\index.js:40:27)
  code: 'ER_ACCESS_DENIED_ERROR',
  errno: 1045,
  sqlMessage: 'Access denied for user \'xxx\'@\'localhost\' (using password: YES)',
  sqlState: '28000',
  fatal: true }
[2017-11-27T10:21:51.741] [ERROR] app - { Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'xxx'@'localhost' (using password: YES)
    at Handshake.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Handshake.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Handshake.js:130:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at Protocol.handshake (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:52:23)
    at PoolConnection.connect (F:\project\project\blog\node_modules\mysql\lib\Connection.js:130:18)
    at Pool.getConnection (F:\project\project\blog\node_modules\mysql\lib\Pool.js:48:16)
    at Promise (F:\project\project\blog\dao\menu.js:11:12)
    at new Promise (<anonymous>)
    at Object.exports.getMenu (F:\project\project\blog\dao\menu.js:9:19)
    at Promise (F:\project\project\blog\routes\index.js:53:42)
    at new Promise (<anonymous>)
    at F:\project\project\blog\routes\index.js:50:27
  code: 'ER_ACCESS_DENIED_ERROR',
  errno: 1045,
  sqlMessage: 'Access denied for user \'xxx\'@\'localhost\' (using password: YES)',
  sqlState: '28000',
  fatal: true }
[2017-11-27T10:21:51.862] [ERROR] app - { Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'xxx'@'localhost' (using password: YES)
    at Handshake.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Handshake.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Handshake.js:130:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at Protocol.handshake (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:52:23)
    at PoolConnection.connect (F:\project\project\blog\node_modules\mysql\lib\Connection.js:130:18)
    at Pool.getConnection (F:\project\project\blog\node_modules\mysql\lib\Pool.js:48:16)
    at Promise (F:\project\project\blog\dao\cms.js:34:12)
    at new Promise (<anonymous>)
    at Object.exports.selectList (F:\project\project\blog\dao\cms.js:11:19)
    at Promise (F:\project\project\blog\routes\index.js:31:41)
    at new Promise (<anonymous>)
    at F:\project\project\blog\routes\index.js:30:27
  code: 'ER_ACCESS_DENIED_ERROR',
  errno: 1045,
  sqlMessage: 'Access denied for user \'xxx\'@\'localhost\' (using password: YES)',
  sqlState: '28000',
  fatal: true }
[2017-11-27T10:21:51.863] [ERROR] app - Something went wrong: { Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'xxx'@'localhost' (using password: YES)
    at Handshake.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Handshake.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Handshake.js:130:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at Protocol.handshake (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:52:23)
    at PoolConnection.connect (F:\project\project\blog\node_modules\mysql\lib\Connection.js:130:18)
    at Pool.getConnection (F:\project\project\blog\node_modules\mysql\lib\Pool.js:48:16)
    at Promise (F:\project\project\blog\dao\cms.js:34:12)
    at new Promise (<anonymous>)
    at Object.exports.selectList (F:\project\project\blog\dao\cms.js:11:19)
    at Promise (F:\project\project\blog\routes\index.js:31:41)
    at new Promise (<anonymous>)
    at F:\project\project\blog\routes\index.js:30:27
  code: 'ER_ACCESS_DENIED_ERROR',
  errno: 1045,
  sqlMessage: 'Access denied for user \'xxx\'@\'localhost\' (using password: YES)',
  sqlState: '28000',
  fatal: true }
[2017-11-27T10:21:51.865] [ERROR] app - { Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'xxx'@'localhost' (using password: YES)
    at Handshake.Sequence._packetToError (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Handshake.ErrorPacket (F:\project\project\blog\node_modules\mysql\lib\protocol\sequences\Handshake.js:130:18)
    at Protocol._parsePacket (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:279:23)
    at Parser.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Parser.js:76:12)
    at Protocol.write (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (F:\project\project\blog\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:125:13)
    at Socket.emit (events.js:221:7)
    at addChunk (_stream_readable.js:265:12)
    at readableAddChunk (_stream_readable.js:252:11)
    --------------------
    at Protocol._enqueue (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:145:48)
    at Protocol.handshake (F:\project\project\blog\node_modules\mysql\lib\protocol\Protocol.js:52:23)
    at PoolConnection.connect (F:\project\project\blog\node_modules\mysql\lib\Connection.js:130:18)
    at Pool.getConnection (F:\project\project\blog\node_modules\mysql\lib\Pool.js:48:16)
    at Promise (F:\project\project\blog\dao\cms.js:78:12)
    at new Promise (<anonymous>)
    at Object.exports.selectListTotal (F:\project\project\blog\dao\cms.js:63:19)
    at Promise (F:\project\project\blog\routes\index.js:41:42)
    at new Promise (<anonymous>)
    at menu (F:\project\project\blog\routes\index.js:40:27)
  code: 'ER_ACCESS_DENIED_ERROR',
  errno: 1045,
  sqlMessage: 'Access denied for user \'xxx\'@\'localhost\' (using password: YES)',
  sqlState: '28000',
  fatal: true }
[2017-11-27T11:52:08.148] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-27T11:54:03.089] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-27T11:54:07.378] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-27T12:03:32.851] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-27T13:39:44.238] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-27T13:40:20.822] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-27T13:41:03.591] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-27T13:44:50.046] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-27T13:44:57.729] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-27T13:46:14.554] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-27T16:27:48.045] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-29T09:12:54.147] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-29T09:12:54.243] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-29T09:12:58.132] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-29T09:12:58.234] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-29T09:22:55.778] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-29T09:23:32.075] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-29T09:44:19.458] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
[2017-11-29T09:45:33.474] [ERROR] app - Something went wrong: { Error: Not Found
    at F:\project\project\blog\app.js:47:15
    at Layer.handle [as handle_request] (F:\project\project\blog\node_modules\express\lib\router\layer.js:95:5)
    at trim_prefix (F:\project\project\blog\node_modules\express\lib\router\index.js:317:13)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:284:7
    at Function.process_params (F:\project\project\blog\node_modules\express\lib\router\index.js:335:12)
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:275:10)
    at F:\project\project\blog\node_modules\express\lib\router\index.js:635:15
    at next (F:\project\project\blog\node_modules\express\lib\router\index.js:260:14)
    at Function.handle (F:\project\project\blog\node_modules\express\lib\router\index.js:174:3)
    at router (F:\project\project\blog\node_modules\express\lib\router\index.js:47:12) status: 404 }
